### PR TITLE
updated to 4.15.0

### PR DESCRIPTION
**Pull request by sadensmol**

### CODE REVIEW

File: build.config [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Path to AIR SDK
-air.sdk = {AIR_SDK_PATH}
+air.sdk = /Applications/Adobe/air22_flex46
```
Comment (ekilah) [inline build.config]: this file should be removed

**Review by ekilah — CHANGES_REQUESTED** [review]
im still having the weird issue we discussed on slack after updating my local setup to air 23...

**Comment by sadensmol** [comment]
@ekilah removed file from the repo.. going to test with air23

**Comment by sadensmol** [comment]
@ekilah tested with air23 and had the same error. I guess air23 is still in beta and FAN wasn't tested with it. Let's wait release of air23.

**Pull request closed without merging**